custom/icds_reports/admin.py

```diff
@@ -1,9 +1,23 @@
 from django.contrib import admin
 
 from custom.icds_reports.models import AggregateSQLProfile
+from custom.icds_reports.models.util import UcrReconciliationStatus
+from custom.icds_reports.tasks import reconcile_data_not_in_ucr
 
 
 @admin.register(AggregateSQLProfile)
 class AggregateSQLProfileAdmin(admin.ModelAdmin):
     list_display = ('name', 'date', 'duration')
     list_filter = ('name', 'date')
+
+
+@admin.register(UcrReconciliationStatus)
+class UcrReconciliationStatusAdmin(admin.ModelAdmin):
+    list_display = ('db_alias', 'day', 'table_id', 'verified_date')
+    actions = ['queue_reconciliation']
+
+    def queue_reconciliation(self, request, queryset):
```

Comment:
This action in the admin is currently the only way to queue these tasks. I thought for a while, but wasn't quite sure how to effectively queue them at this time. I think the factors we need to balance are:

* load on the shard database (wouldn't want to run all at same time during the IST day)
* load on the UCR database (wouldn't want to run all at same time during aggregation)
* pillow backlog (don't want to check yesterday's numbers if we already know we're behind)

Some options:

* All non-verified records can be queued at some time
* All non-verified records from two days ago
* All non-processed records

Anyways, For now I think I'm going to just create an initial set manually (there is a periodic task that creates new records  each day) and run them manually as I monitor db load